features/onboarding/Privy/PrivyPhoneEntry.tsx

```diff
@@ -7,19 +7,20 @@ import {
   textSecondaryColor,
 } from "@styles/colors";
 import { spacing } from "@theme/spacing";
-import { AsYouType, CountryCode } from "libphonenumber-js";
+import type { CountryCode } from "libphonenumber-js";
+import { AsYouType } from "libphonenumber-js";
```

Comment:
Just curious - Did these get split because of the new linter? Or something you caught yourself?

Reply:
Oh sorry never answered. ESLint was giving me an error for that